scenarios/GPT-4V/video_by_manual_sampling/video_chatcompletions_example_restapi.ipynb

```diff
@@ -168,7 +168,7 @@
    "id": "feabcd3f",
    "metadata": {},
    "source": [
-    "### Call GPT-4 Turbo with Vision API"
+    "### Sample frame and call GPT-4 Turbo"
```

Comment:
Should be "Sample frames"?

Also need to do similar change in video_chunk_chatcompletions_example_restapi.ipynb